CMakeLists.txt

```diff
@@ -235,6 +238,10 @@ IF (MSVC)
 	IF(RTTR_ENABLE_WERROR)
 		ADD_DEFINITIONS(/WX) # warning = error
 	ENDIF()
+	# Link time code generation (Required for bzlib) which also disables incremental linking
+	ADD_FLAGS(CMAKE_EXE_LINKER_FLAGS /LTCG)
+	FORCE_ADD_FLAGS(CMAKE_EXE_LINKER_FLAGS_DEBUG /INCREMENTAL:NO)
```

Comment:
don't do that for debug! do not set LTCG if not necessary. incremental linking is extremly important if you do  live-debugging (with disabling incremental, no edit+continue works anymore)

Reply:
The bzip2 in the contribution dir forces this. But maybe we should just update that one instead